### PR TITLE
fix: clean openshift config map

### PR DESCRIPTION
**Pull request by ia3andy**
Add optional to:
    - LAUNCHER_BACKEND_URL
    - LAUNCHER_KEYCLOAK_CLIENT_ID
    - LAUNCHER_TRACKER_SEGMENT_TOKEN
    - LAUNCHER_SENTRY_DSN

### CODE REVIEW

File: config/webpack.prod.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var commonConfig = require('./webpack.common.js');
 var helpers = require('./helpers');
 
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-const LAUNCHER_BACKEND_URL = process.env.LAUNCHER_BACKEND_URL || '/launch/api';
+const LAUNCHER_BACKEND_URL = process.env.LAUNCHER_BACKEND_URL;
```
Comment (gastaldi) [inline config/webpack.prod.js]: I think Minishift installations do not specify this parameter, but I may be wrong

Comment (ia3andy) [reply config/webpack.prod.js]: where is the minishift setup?

Comment (ia3andy) [reply config/webpack.prod.js]: so I'll make it optional then

Comment (gastaldi) [reply config/webpack.prod.js]: Try installing the openshift template and see if this parameter is set in the frontend deploymentconfig

**Comment by gastaldi** [comment]
Don't forget to sync with the template in the Openshift templates repository, there is a script that does that

**Pull request merged**